### PR TITLE
Dlg transpose text

### PR DESCRIPTION
**Pull request by lilyclements**
@africanmathsinitiative/developers this is ready for review

Changes:
- Updated the SetDefaults()
- Fixed developer error for a disabled control
- Sorted layout

### CODE REVIEW

**Comment by lilyclements** [comment]
@africanmathsinitiative/developers This is still ready for review. This is fixing a developer error so needs to go into tomorrow's version

**Comment by stevekogo** [comment]
I am going to review this myself.

**Comment by stevekogo** [comment]
What is your view on overwriting an existing data frame? Should be allow this to happen or should we give a warning?

**Comment by lilyclements** [comment]
We should give a warning, however, I believe the plan is to put this in the ucrSave and it can apply to both columns and data frames

**Comment by stevekogo** [comment]
That should then be documented probably as an issue.

**Comment by lilyclements** [comment]
Okay, good point. I'll write an issue up on it within the hour, thanks!

**Comment by lilyclements** [comment]
@stevekogo did you review this in the end?

**Comment by stevekogo** [comment]
@lilyclements I am going to revisit this.

File: instat/dlgTransposeColumns.vb [hunk 1]
```diff
@@ -49,29 +49,26 @@ Public Class dlgTransposeColumns
         ucrNewDataframe.SetLabelText("New Data Frame Name:")
 
         'chkbox
-        'checkbox not implemented as it wasn't on the old version
         'ucrChkNameNewColumns.SetParameter(New RParameter(""))
```
Comment (stevekogo) [inline instat/dlgTransposeColumns.vb]: Just tidy up the code in this section and clarify why the checkbox is disabled. Otherwise, this is working fine.

Comment (lilyclements) [inline instat/dlgTransposeColumns.vb]: Great, I've fixed this

**Comment by lilyclements** [comment]
@stevekogo I've made the necessary amendments and sorted parameter positioning (I realised that was not yet implemented). This is ready for a re-review.

**Comment by stevekogo** [comment]
@dannyparsons This is now ready for you.

**Pull request merged**